dashboard/index.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+exec('sudo docker ps -q', $containers);
+$data = [];
+
+foreach ($containers as $container) {
+    $containerInspectCmd = "sudo docker inspect --format='{{json .}}' " . $container;
+    $containerInfo = json_decode(shell_exec($containerInspectCmd));
+    $envVariableArray = preg_grep('/VIRTUAL_HOST=.*$/', $containerInfo->Config->Env);
+    $virtualHostArray = [];
+    if (!empty($envVariableArray)) {
+        $virtualHostEnv = $envVariableArray[0];
+        $virtualHosts = explode('=', $virtualHostEnv)[1];
+        $virtualHostArray = [$virtualHosts];

```

Comment:
what are you doing here?

Reply:
Initialising the virtualHostArray with the VIRTUAL_HOST env variable. This remains unchanged if there aren't multiple entries in VIRTUAL_HOST